Library/DiscUtils.Ntfs/Bitmap.cs

```diff
@@ -201,6 +201,16 @@ public byte GetByte(long index)
             return 0;
         }
 
+        public int GetBytes(byte[] buffer, long index, int count)
```

Comment:
This signature was actually a bad copy of `GetByte`. I've added offset made both internal and reordered the parameters to be more clear.